includes/zigbee2mqtt/configuration.yaml

```diff
@@ -8,7 +8,7 @@ mqtt:
   user: "%user%"
   password: "%password%"
 serial:
-  port: "/dev/serial/%adapter"
```

Comment:
is %adapter correct or is the % at the end missing ? The other placeholders have it.

(note reviewing code is not about testing if your solution works)

Reply:
The missing % at the end comes from the original code. I will test if adding a leading % makes any difference.